apps/web/lib/language/tests/language.test.ts

```
@@ -0,0 +1,140 @@
+  projectCache: { revalidate: vi.fn() },
+}));
+vi.mock("@/lib/survey/cache", () => ({
+  surveyCache: { revalidate: vi.fn() },
+}));
+
+const fakeProject = {
+  id: mockProjectId,
+  environments: [{ id: "env1" }, { id: "env2" }],
+} as TProject;
+
+const testInputValidation = async (service: Function, ...args: any[]): Promise<void> => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)